go/mysqlconn/replication/binlog_event.go

```diff
@@ -297,6 +243,17 @@ func (b *Bitmap) Bit(index int) bool {
 	return b.data[byteIndex]&bitMask > 0
 }
 
+// Set sets the given boolean value.
+func (b *Bitmap) Set(index int, value bool) {
```

Comment:
In the last review, I was only reading the bitmaps, not generating them, as I was missing the generation of the RBR events.

Yes, coverage on this file in 100%, but it's because it's tested indirectly by the individual packet methods. I'll add more tests in subsequent changes.